.github/workflows/unit_test.yml

```
@@ -0,0 +1,86 @@
+name: Unit Tests
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches-ignore:
+      - master
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+env:
+  DEFAULT_PYTHON: '3.11'
+
+jobs:
+  tests:
+    name: Run unit tests (Python 3.11, Pydantic v1 on ubuntu-latest)
```

```suggestion
    name: Run unit tests
```

Пока у нас нет matrix build, думаю нет смысла выводить текущую версию Python. И Pydantic v1 тут не используется